### DISCUSSION TITLE
Backend.ai Web-ui 설치 질문드립니다.

### DISCUSSION THREAD

**Discussion by TeusEE** [lablup/contribution-academy-2021#66] [Q&A] [answered]
**Describe the problem**
Backend.ai 설치 완료 후 실행 테스트까지 완료하였습니다.
챌린지 3의 2번째 Web Ui 실행하기 관련해서, 해당 부분 과제 자체가 무엇인지 잘 이해가 안됩니다.
Backend.ai -webui github에서 사용자 메뉴얼을 보면

webui stand alone 프로그램을 사용하면 된다고 나와있구,

github 자체를 보면 
Set-up Guide에 Backend.ai webserver의 하위 컴포넌트(refered?) 이고, Backend.ai webserver가 설치되어 있을 경우 이미 최신 배포된 안정화 버전의 web-ui 를 설치가 되었다고 이해했습니다.

개인적으로는 github만 봐서는 web-ui를 어떻게 실행해야하는지 전혀 감이안오는데, 이부분 어떻게해야할지 조언좀 부탁드립니다!

**To Reproduce**

**Screenshots**

**Environment Description**
OS: Windows10 WSL2 Ubuntu
Current Shell: bash

**Comment by lizable** [comment, marked as answer]
@TeusEE
[`backend.ai-webserver`](https://github.com/lablup/backend.ai-webserver) indeed serves WebUI as a compiled(build) component, but in this task you need to install and execute WebUI separately, not using [built WebUI in webserver](https://github.com/lablup/backend.ai-app/tree/b3b89a908c0116309ddf2169c47b857777193170).
Probably [this link](https://github.com/lablup/backend.ai-webui#developing--testing-without-bundling) gives you a clue.
And please remember those npm commands written above the link need to be executed separately.
Using `tmux` in ttyd will be lots of help.
